.github/CONTRIBUTING.md

```diff
@@ -1,151 +1,52 @@
-The exact request to be triggered depends on what you are doing. If investigating functional test failures locally, then the tests logs should contain the url it hit to trigger scenarios.
+### Unit Tests
 
-Dependency Collector tests deploy the test apps, along with dependencies (Fake Ingestion, SQL etc) to Docker containers inside same Docker virtual network, so that apps can access the dependencies with their names. However, if the test apps are deployed to IIS or IISExpress, then they are outside the Docker virtual network of dependencies, and so it won't be able to access dependencies without using their IP Address. This is a Docker for windows limitation, and could be fixed in future.
-Until then, the test app need to address the dependencies using their IP Address. Instead of manually finding IP Addresses and replacing containers names with IP Address, its easy to just run the following script.
-This uses Docker commands to determine the IP Addresses, and replaces them in the necessary configs.
-"<repo-root>\bin\Debug\Test\E2ETests\E2ETests\replacecontainernamewithip.ps1"
+Unit tests can be run in either the Visual Studio Test Exploror or via .NET CLI `dotnet test` ([link](https://docs.microsoft.com/dotnet/core/tools/dotnet-test)).
 
-Following pre-requisite is needed to deploy to IIS locally.
-* IIS (Make sure Internet Information Services > World Wide Web Services > Application Development Features > ASP.NET 4.6 is enabled)
+This repo also has some Integration Tests which cannot be run as a standalone DLL and must be run in the context of their project (*.csproj).
+For more information please visit [Integration tests in ASP.NET Core](https://docs.microsoft.com/aspnet/core/test/integration-tests).
 
+See also our [GitHub Workflows](/.github/workflows) for examples of running these tests.
 
 ## Debugging the SDK in general (How to test Application Insights from local build in any Test App)
 
-* Build the project using ```buildDebug.cmd```. If you build using the solution (*.sln) all required depenencies will also be built.
-* If the build was successful, you'll find that it generated NuGet packages in "<repository root>\..\bin\Debug\NuGet". You can set this directory as a NuGet Repository to consume within your applications.
-* Create a web application project to test the SDK on, and install the Microsoft.ApplicationInsights.Web NuGet package from the above directory.
-* From your web application, open the .cs file you want your breakpoint in and set it
-* Run your web application. Your breakpoints should be hit now when your web application triggers them.
+- Build any solution using `dotnet build`. All required depenencies will also be built.
+  - If the build was successful, you'll find that it generated NuGet packages in "repository root\bin\Debug\NuGet". 
+    You can set this directory as a NuGet Repository to consume within your applications.
```

Comment:
these instructions were originally important, as all sdks were in dif.. repos, and were having nuegt references with each other. With full project reference now, these instructions are irrelevant.

Reply:
I'm okay leaving these instructions here because I don't think this is common knowledge.
If you feel strongly about this I can remove it. :)
Will sync with you offline.